src/admin/content-block/helpers/generators/defaults.ts

```diff
@@ -14,27 +16,43 @@ export const FORM_STATE_DEFAULTS = (
 });
 
 export const CONTENT_BLOCK_FIELD_DEFAULTS = () => ({
-	...BACKGROUND_COLOR_FIELD(),
+	background: BACKGROUND_COLOR_FIELD(),
 });
 
 // Recurring fields
 export const BACKGROUND_COLOR_FIELD = (label: string = 'Achtergrondkleur') => ({
-	background: {
-		label,
-		editorType: ContentBlockEditor.ColorSelect,
-		editorProps: {
-			options: BACKGROUND_COLOR_OPTIONS,
-			defaultValue: BACKGROUND_COLOR_OPTIONS[0],
-		},
+	label,
+	editorType: ContentBlockEditor.ColorSelect,
+	editorProps: {
+		options: BACKGROUND_COLOR_OPTIONS,
+		defaultValue: BACKGROUND_COLOR_OPTIONS[0],
 	},
 });
 
 export const ALIGN_FIELD = (label: string = 'Uitlijning') => ({
-	align: {
-		label,
-		editorType: ContentBlockEditor.AlignSelect,
-		editorProps: {
-			options: ALIGN_OPTIONS,
-		},
+	label,
+	editorType: ContentBlockEditor.AlignSelect,
+	editorProps: {
+		options: ALIGN_OPTIONS,
+	},
+});
+
+export const TEXT_FIELD = (
+	label: string = 'Tekst',
+	emptyFieldValidatorMessage: string = 'Tekstveld moet verplicht ingevuld zijn.'
```

Comment:
type string is overbodig

https://stackoverflow.com/questions/46799828/tslint-type-trivially-inferred-why-is-it-bad-practice-to-include-the-type-he